test/unit/builtins/string.spec.ts

```diff
@@ -54,24 +55,46 @@ test("string index (side effect)", () => {
     `.expectToMatchJsResult();
 });
 
-test.each([
-    { inp: "hello test", searchValue: "", replaceValue: "" },
-    { inp: "hello test", searchValue: " ", replaceValue: "" },
-    { inp: "hello test", searchValue: "hello", replaceValue: "" },
-    { inp: "hello test", searchValue: "test", replaceValue: "" },
-    { inp: "hello test", searchValue: "test", replaceValue: "world" },
-    { inp: "hello test", searchValue: "test", replaceValue: "%world" },
-    { inp: "hello test", searchValue: "test", replaceValue: "." },
-    { inp: "hello %test", searchValue: "test", replaceValue: "world" },
-    { inp: "hello %test", searchValue: "%test", replaceValue: "world" },
-    { inp: "hello test.", searchValue: ".", replaceValue: "$" },
-    { inp: "hello test", searchValue: "test", replaceValue: () => "a" },
-    { inp: "hello test", searchValue: "test", replaceValue: () => "%a" },
-    { inp: "aaa", searchValue: "a", replaceValue: "b" },
-])("string.replace (%p)", ({ inp, searchValue, replaceValue }) => {
-    util.testExpression`"${inp}".replace(${util.formatCode(searchValue, replaceValue)})`.expectToMatchJsResult();
+describe.each(["replace", "replaceAll"])("string.%s", method => {
+    const testCases = [
+        { inp: "hello test", searchValue: "", replaceValue: "" },
+        { inp: "hello test", searchValue: "", replaceValue: "_" },
+        { inp: "hello test", searchValue: " ", replaceValue: "" },
+        { inp: "hello test", searchValue: "hello", replaceValue: "" },
+        { inp: "hello test", searchValue: "test", replaceValue: "" },
+        { inp: "hello test", searchValue: "test", replaceValue: "world" },
+        { inp: "hello test", searchValue: "test", replaceValue: "%world" },
+        { inp: "hello test", searchValue: "test", replaceValue: "." },
+        { inp: "hello %test", searchValue: "test", replaceValue: "world" },
+        { inp: "hello %test", searchValue: "%test", replaceValue: "world" },
+        { inp: "hello test.", searchValue: ".", replaceValue: "$" },
+        { inp: "aaa", searchValue: "a", replaceValue: "b" },
+    ];
+    describe("string replacer", () => {
+        test.each(testCases)("%p", ({ inp, searchValue, replaceValue }) => {
```

Comment:
```suggestion
    test.each(testCases)("string replacer %p", ({ inp, searchValue, replaceValue }) => {
```
You shouldn't nest describes, and in this case I don't see why it is needed in the first place as there is only one test inside.